ubiquity/pay.ubq.fi
# PR: offer new fallback intl mastercard SKU 18732

Resolves #349

Other than the addition of the new mastercard 18732, it has following improvements.
- Added some trusted and reliable RPCs back as  a backup for backend if RPC handler crashes. It goes crazy sometimes. e.g. couldn't resolve DNS of an rpcs and crashes the whole process 
- show product SKU on UI with the card, will be helpful for troubleshooting if there is a problem. 

## How to test?

- fill .env vars (also setup wallets, funding wallet, approval, beneficiary wallet with uusd token for testing and xdai for gas)
- fill wrangler.toml with production keys of Reloadly
- yarn build && yarn start
- your IP location should be South Korea or any other country that support 18732 but doesn't have their country specific card available. 
- if you want the permit amount to go to your wallet for testing instead of the treasury, change the treasury address at /shared/constants.ts

NOTE: All orders with production API keys of Reloadly will deduct the real balance from Reloadly. 

### QA

https://github.com/user-attachments/assets/7a2c9e0c-428c-4889-b3c5-7c70d2af01f6


#### /ubiquity-dollar Location: South Korea

![Screenshot_20241213_010128](https://github.com/user-attachments/assets/58658322-4cc4-4dd7-ada7-9e64a2e87ff0)


#### /ubiquity-dollar Location: Germany

![Screenshot_20241212_165301](https://github.com/user-attachments/assets/ae0d6cd8-e722-4fe4-80fb-a7256788fbdc)
